build/logs/clover.xml

```diff
@@ -0,0 +1,333 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think we don't need to the `clover.xml` under the Git version control.

Reply:
yep I think we should has the build folder in the gitignore